subgraphs/rocket-pool/src/updaters/financialMetrics.ts

```diff
@@ -0,0 +1,461 @@
+import {
+  Address,
+  BigInt,
+  BigDecimal,
+  ethereum,
+  log,
+} from "@graphprotocol/graph-ts";
+import { bigIntToBigDecimal } from "../utils/numbers";
+import { getOrCreateProtocol } from "../entities/protocol";
+import { getOrCreatePool } from "../entities/pool";
+import {
+  FinancialsDailySnapshot,
+  PoolDailySnapshot,
+  PoolHourlySnapshot,
+} from "../../generated/schema";
+import { getOrCreateToken } from "../entities/token";
+import {
+  BIGDECIMAL_ZERO,
+  ETH_ADDRESS,
+  RETH_ADDRESS,
+  SECONDS_PER_DAY,
+  SECONDS_PER_HOUR,
+  RPL_ADDRESS,
+} from "../utils/constants";
+
+const PROTOCOL_ID = RETH_ADDRESS;
+
+export function updateProtocolAndPoolRewardsTvl(
+  blockNumber: BigInt,
+  blockTimestamp: BigInt,
+  rewardAmount: BigInt
+): void {
+  const pool = getOrCreatePool(blockNumber, blockTimestamp);
+  const protocol = getOrCreateProtocol();
+
+  const inputTokenBalances: BigInt[] = [];
+  inputTokenBalances.push(rewardAmount);
+  inputTokenBalances.push(pool.inputTokenBalances[1]);
```

Comment:
Where is the inputTokenBalance set for ether? I can't find that

Reply:
that does rewards, the next function does eth. Line 72